aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3111,7 +3122,7 @@ std::tuple<Tensor&, Tensor&, Tensor&> svd_out(const Tensor& self, bool some, boo
   return std::tie(U, S, V);
 }
 
-std::tuple<Tensor, Tensor, Tensor> svd(const Tensor& self, bool some, bool compute_uv) {
+std::tuple<Tensor, Tensor, Tensor> svd(const Tensor& self, bool some, bool compute_uv, c10::optional<c10::string_view> driver) {
```

Comment:
I would suggest not to add a `driver` kwarg for this function, as it is deprecated and we will (hopefully) remove it soon-ish (cc @mruberry).
I would simply default this driver to be `gesvdj` and run with it.